### PR TITLE
RGO 2017

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: launch/start.launch [hunk 1]
```diff
@@ -33,10 +33,13 @@
                 -s $(env ROBOT_BRINGUP_PATH)/parameters/hardware/actuators/rgb_lights.ops
                 "/>
                 <arg name="NAME" value ="hardware"/>
-                <arg name="LOG_LEVEL" value="Info"/>
+                <arg name="LOG_LEVEL" value="Warning"/>
```
Comment (Timple) [inline launch/start.launch]: Lets leave this Info and put all unnecessary logging to log_debug

File: launch/start.launch [hunk 2]
```diff
@@ -33,10 +33,13 @@
...
                 <arg name="DEBUG" value="$(arg debug)"/>
                 <arg name="machine" value="amigo1" />
             </include>
+            
+            <!-- BODYPART RESETTER ROBOCUP HACK -->
+            <node name="spindle_resetter" pkg="robot_launch_files" type="bodypart_resetter.py" args="amigo spindle 1"/>
```
Comment (Timple) [inline launch/start.launch:42]: This is ok to leave in. Can be removed when https://github.com/tue-robotics/tue_robocup/issues/257 is solved

**Review by Timple — DISMISSED** [review]
I did not (carefully) review changes in challenges. So another review for those might be useful.

File: parameters/navigation/global_costmap.yaml [hunk 3]
```diff
@@ -27,9 +27,10 @@ ed_occupancy_grid:
 
 recent_obstacles:
     occupied_to_default_time: 2.0
-    observation_sources: base_laser torso_laser top_kinect
+    # observation_sources: base_laser torso_laser top_kinect
+    observation_sources: base_laser top_kinect
```
Comment (MatthijsBurgh) [inline parameters/navigation/global_costmap.yaml:31]: We should discuss the use of the torso laser in the costmaps

File: parameters/navigation/local_costmap.yaml [hunk 4]
```diff
@@ -48,9 +48,10 @@ ed_occupancy_grid:
 
 recent_obstacles:
     occupied_to_default_time: 2.0
-    observation_sources: base_laser torso_laser top_kinect
+    # observation_sources: base_laser torso_laser top_kinect
+    observation_sources: base_laser top_kinect
     base_laser:  {data_type: LaserScan, sensor_frame: /amigo/base_laser, topic: /amigo/base_laser/scan, observation_persistence: 0.0, inf_is_valid: true, obstacle_range: 10.0, marking: true, clearing: false, ray_trace_range: 10.0}
-    torso_laser:  {data_type: LaserScan, sensor_frame: /amigo/torso_laser, topic: /amigo/torso_laser/scan, observation_persistence: 0.0, inf_is_valid: true, obstacle_range: 10.0, marking: true, clearing: false, ray_trace_range: 10.0}
+    # torso_laser:  {data_type: LaserScan, sensor_frame: /amigo/torso_laser, topic: /amigo/torso_laser/scan, observation_persistence: 0.0, inf_is_valid: true, obstacle_range: 10.0, marking: true, clearing: false, ray_trace_range: 10.0}
```
Comment (MatthijsBurgh) [inline parameters/navigation/local_costmap.yaml:54]: We should discuss the use of the torso laser in the costmaps

**Review by LoyVanBeek — APPROVED** [review]
I trust in you that this actually works

**Pull request merged**